### PR TITLE
added new style underline to blog titles

### PR DESCRIPTION
**Pull request by gregdorward**
Small change to apply the new badgerOnWhite colour to the blog title underline

### CODE REVIEW

**Comment by gregdorward** [comment]
@dstodolny - here's your chance for revenge for the other 2 tickets :)

**Comment by chris-brookes** [comment]
Only need the review now @dstodolny .

File: styles/_blog.less [hunk 1]
```diff
@@ -93,7 +93,7 @@ body:not(.button-style-default) .sqs-editable-button, body:not(.button-style-def
   text-decoration: none;
 }
 .entry-title a:hover {
-  .redUnderline();
+  border-bottom: 2px solid #e5051a;
```
Comment (dstodolny) [inline styles/_blog.less]: `border-bottom: 2px solid @badgerRedOnWhite;`

**Comment by chris-brookes** [comment]
Colour still looks good on the site.

**Comment by gregdorward** [comment]
@dstodolny could we get this merged into master please? I'll pull the latest into my other branch then, which should be good to go soon

**Pull request merged**